RELEASE_NOTES.rst

```diff
@@ -41,6 +41,8 @@ Significant Changes
 New Features
 """"""""""""
 - AIP-58: Add Airflow ObjectStore (AFS) (`AIP-58 <https://github.com/apache/airflow/pulls?q=is%3Apr+is%3Amerged+label%3AAIP-58+milestone%3A%22Airflow+2.8.0%22>`_)
+- Add XCom tab to Grid (#35719)
+- Add a public interface for custom ``weight_rule`` implementation (#35210)
```

Comment:
We decided to remove it

Reply:
Yeah see: https://github.com/apache/airflow/pull/36065#issuecomment-1840504204

Reply:
Reverted